### ISSUE TITLE
[Feature Request] Add .ini output for the website

### ISSUE THREAD

**Issue by TheraNinjaCat** [legoduded/palworld-worldoptions#19]
It's possible to import from the ini, however if I edit the config using the website there's now a discrepancy between the original ini file and the resulting WorldOptions file. A solution for this would be to add an option to also output the settings in ini format to replace the original ini file. A different solution would be to support importing the WorldOptions file as well as ini.

**Comment by fermioff** [comment]
First off, thank you for this tool!
I've just started hosting a server this week for myself and friends and @TheraNinjaCat 's request is exactly what I came here looking for. It would be great to have an output txt/ini file in plaintext to help in keeping a changelog and to keep track of multiple world states . 
A basic changelog can be kept manually, of course, yet it would be great to have the txt/ini files for the purpose of maintaining a catalog while also allowing us to more feasibly cross-reference each world state. 
Examples where this output would facilitate managing a server:
1) We'd like to test the numerous server settings and how certain varying combinations might work together; 
2) We'd like to create "event" style world states (e.g. long nights, super deterioration) and a txt/ini output would make this much more manageable
